README.md

```diff
@@ -0,0 +1,67 @@
+
+# fv3atm
+
+This repository contains a driver and key subcomponents of the
+atmospheric component of the NOAA's [Unified Forecast System
+(UFS)](https://ufscommunity.org/) weather model.
+
+The subcomponents include:
+
+ - The Finite-Volume Cubed-Sphere (FV3) dynamical core, originally
+from the [Geophysical Fluid Dynamics
+Laboratory](https://www.gfdl.noaa.gov/).
+ - The Common Community Physics Package (CCPP) supported by the
+   [Developmental Testbed Center
+   (DTC)](https://dtcenter.org/community-code/common-community-physics-package-ccpp),
+   including:
+   - [CCPP Framework](https://github.com/NCAR/ccpp-framework).
+   - [CCPP Physics](https://github.com/NCAR/ccpp-physics)
+ - wrapper code to call [UFS stochastic
+   physics](https://stochastic-physics.readthedocs.io/en/latest/)
+ - The io code handles netCDF I/O.
+ - The cpl coupler code connects the different components and allows
+   them to communicate.
+
+## Prerequisites
+
+This package requires the following
+[NCEPLIBS](https://github.com/NOAA-EMC/NCEPLIBS) packages:
```

Comment:
I am not sure if this means the packages required by fv3atm or by fv3atm and its subcomponents

Reply:
Let us list them both. I have listed the dependencies of fv3atm. What are the dependencies of its subcomponents?